src/local-dem-manager.ts

```diff
@@ -55,26 +47,19 @@ export class LocalDemManager implements DemManager {
   maxzoom: number;
   timeoutMs: number;
   loaded = Promise.resolve();
-  decodeImage: (
-    blob: Blob,
-    encoding: Encoding,
-    abortController: AbortController,
-  ) => Promise<DemTile> = decodeImage;
+  decodeImage: DecodeImageFunction;
+  getTile: GetTileFunction;
 
-  constructor(
-    demUrlPattern: string,
-    cacheSize: number,
-    encoding: Encoding,
-    maxzoom: number,
-    timeoutMs: number,
-  ) {
-    this.tileCache = new AsyncCache(cacheSize);
-    this.parsedCache = new AsyncCache(cacheSize);
-    this.contourCache = new AsyncCache(cacheSize);
-    this.timeoutMs = timeoutMs;
-    this.demUrlPattern = demUrlPattern;
-    this.encoding = encoding;
-    this.maxzoom = maxzoom;
+  constructor(options: DemManagerInitizlizationParameters) {
```

Comment:
Looks like this change broke the benchmark script. Can you update the LocalDemManager constructor call there? You can test using `npm run build` and `node benchmark.js`

Reply:
Should be fixed now. I would recommend changing the settings for CI run to allow veteran github users to activate the CI.